### PR TITLE
Add dredd testing to Sessions API

### LINKED ISSUE #3900: Add sessions api to dredd testing
parent issue: #3870

### PR DESCRIPTION
**Pull request by poush**
Fixes #3900
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The unit tests pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

#### Short description of what this resolves:
Added dredd testing to Sessions API

### CODE REVIEW

File: app/api/sessions.py [hunk 1]
```diff
@@ -59,8 +58,8 @@ def validate_date(self, data, original_data):
     ends_at = fields.DateTime(required=True)
     language = fields.Str()
     slides_url = fields.Url(attribute='slides')
-    videos_url = fields.Url(attribute='videos')
-    audios_url = fields.Url(attribute='audios')
+    videos_url = fields.Url(attribute='video')
```
Comment (niranjan94) [inline app/api/sessions.py]: Why is it called videos_url ? It's just video_url ... And also update the db columns accordingly please.

Comment (niranjan94) [inline app/api/sessions.py]: Same for audio_url.

File: app/api/sessions.py [hunk 2]
```diff
@@ -59,8 +58,8 @@ def validate_date(self, data, original_data):
     ends_at = fields.DateTime(required=True)
     language = fields.Str()
     slides_url = fields.Url(attribute='slides')
```
Comment (niranjan94) [inline app/api/sessions.py]: Rename db field also to `slides_url`

**Comment by shubham-padia** [comment]
@niranjan94 please review

**Pull request merged**